core/storage/exploration/gae_models.py

```diff
@@ -818,6 +863,11 @@ def get_recently_published(cls, limit):
             -ExpSummaryModel.first_published_msec
         ).fetch(limit)
 
+    @staticmethod
+    def get_export_policy():
```

Comment:
Yes, it does. (The docstring is wrong.) In general you can't just write this for models that clearly contain user_ids by inspection.

However, the data here is already included in the ExplorationModel export (please verify) and so we don't need to export this one as well.

Reply:
I see ratings information in the ExpSummaryModel, but I don't see this in the ExplorationModel. Am I missing something here?

Reply:
Probably :) I think ratings is stored in ExpUserDataModel, can you check?

Reply:
Yup. Updated.